projects/revest/index.js

```diff
@@ -9,13 +9,35 @@ const {
   transformPolygonAddress,
 } = require("../helper/portedTokens.js");
 
+const { ethers } = require("ethers");
+const { utils } = require("@project-serum/anchor");
```

Comment:
I dont think this is used in the code

Reply:
it does ethers.utils.formatEther(txn.args.amountTokens)

As for the anchor i'm not sure about that, rob added that so I just left it in.

Reply:
ethers yes, but I am guessing utils was added by mistake

Reply:
Yes, utils can be removed, meant to reference ethers.utils